<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editChar</class>
 <widget class="QDialog" name="editChar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Character</string>
  </property>
  <widget class="QLineEdit" name="Name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="NewName">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>New Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Level:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="LevelSpin">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>44</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
  </widget>
  <widget class="QComboBox" name="RaceBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="ClassBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="AlignBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Strength:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Dexterity:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Constitution:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Intelligence:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Wisdom:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Charisma:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="StrSpin">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>44</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>8</number>
   </property>
   <property name="maximum">
    <number>40</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="DexSpin">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>44</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>8</number>
   </property>
   <property name="maximum">
    <number>40</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="ConSpin">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>44</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>8</number>
   </property>
   <property name="maximum">
    <number>40</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="IntSpin">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>44</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>8</number>
   </property>
   <property name="maximum">
    <number>40</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="WisSpin">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>44</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>8</number>
   </property>
   <property name="maximum">
    <number>40</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="ChaSpin">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>44</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>8</number>
   </property>
   <property name="maximum">
    <number>40</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>HP:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="HPSpin">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>44</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PosPerson">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="NeuPerson">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="NegPerson">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>20</width>
     <height>171</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Personality</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ApplyButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DelButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>410</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
